tasks/options/postcss.js

```diff
@@ -11,7 +11,7 @@ module.exports = function(config) {
       },
 
       processors: [
-        require('autoprefixer')({browsers: 'last 3 versions'}), // add vendor prefixes
+        require('autoprefixer')({browsers: ['last 3 versions', 'safari >= 4']}), // add vendor prefixes
```

Comment:
This workaround is recommended in related [phantomjs issue](https://github.com/ariya/phantomjs/issues/14365) and it works like expected.